### PR TITLE
torch.jit doc link for nvfuser readme.md

### PR DESCRIPTION
**Pull request by jjsjann123**
adding a quick link to nvfuser README.md in jit doc

Note that for 1.12 release, we probably want to have the link pointed to the doc in the release code base. I don't know if we have a tag for 1.12 release candidate yet, so we might want to update that.

### CODE REVIEW

File: docs/source/jit.rst [hunk 1]
```diff
@@ -874,6 +874,11 @@ now supported.
             b = 2
         return x, b
 
+Fusion Backends
+~~~~~~~~~~~~~~~
+There are a couple of fusion backends available to optimize TorchScript execution. Along with the default NNC which handles cpu as well as gpu fusion, there is nvfuser which targets GPU fusion and here's a link to their documentation for usage & debug: https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/codegen/cuda/README.md
```
Comment (davidberard98) [inline docs/source/jit.rst:879]: nit: suggestion on wording below (plus a link)

```
There are a couple of fusion backends available to optimize TorchScript execution. The default is NNC, which can perform fusions for both CPU and GPU. Another alternative is NVFuser, which targets GPU fusion. See the  `NVFuser documentation <https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/codegen/cuda/README.md>` for more details on usage and debugging.
```

**Comment by jjsjann123** [comment]
Looks like there's an error with build doc: 

pull / linux-docs / build-docs (python) (pull_request)  https://github.com/pytorch/pytorch/runs/6515553552?check_suite_focus=true

Don't think it's related.. Meanwhile my local build of docs also shows properly updated jit page.

**Comment by jjsjann123** [comment]
@davidberard98 I'm rescuing this PR to update the link in readme to point to the right doc.

**Review by davidberard98 — APPROVED** [review]

**Pull request closed without merging**